src/rpc/mining.cpp

```diff
@@ -98,7 +99,7 @@ static UniValue getnetworkhashps(const JSONRPCRequest& request)
     return GetNetworkHashPS(!request.params[0].isNull() ? request.params[0].get_int() : 120, !request.params[1].isNull() ? request.params[1].get_int() : -1);
 }
 
-UniValue generateBlocks(std::shared_ptr<CReserveScript> coinbaseScript, int nGenerate, uint64_t nMaxTries, bool keepScript)
+static UniValue generateBlocks(CScript& coinbase_script, int nGenerate, uint64_t nMaxTries)
```

Comment:
Just out of curiosity, how come the use of mixed casing? 

`coinbase_script`, `nMaxTries` all in the same function?

Reply:
> Variable (including function arguments) and namespace names are all lowercase, and may use _ to separate words (snake_case).
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c

 > Patchsets should always be focused. For example, a pull request could add a feature, fix a bug, or refactor code; but not a mixture.
https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md